src/valid/analyzer.rs

```diff
@@ -675,12 +675,17 @@ impl FunctionInfo {
                 requirements: UniformityRequirements::empty(),
             },
             E::Math {
-                arg, arg1, arg2, ..
+                fun: _,
+                arg,
+                arg1,
+                arg2,
+                arg3,
```

Comment:
Nice catch. I think I've seen several of these fixed recently. I did a quick look and I think this must the last - the other places that use `Math { .. }` are sites that can't produce results at all without the arguments, so `..` is probably okay there.